packages/@pollyjs/adapter-node-http/src/index.js

```diff
@@ -73,6 +73,14 @@ export default class HttpAdapter extends Adapter {
         // dealing with json content to convert it back to a string.
         if (body && typeof body !== 'string') {
           body = JSON.stringify(body);
+        } else if (
+          req.headers['content-type'] &&
```

Comment:
This interceptor has access to the `content-type` header, so just check it here: if it's a multipart form request, transform the `body`.

Reply:
Should this test that body is a buffer?

Reply:
:+1: Great idea, will update it.